ui/src/app/pages/login/account-creation-images.tsx

```diff
@@ -0,0 +1,15 @@
+export interface PageImages {
+  backgroundImgSrc: string;
+  smallerBackgroundImgSrc: string;
+}
+
+export const pageImages = {
```

Comment:
It'll be nice to have consistent class and const name. Also since it contains login page technically its not account creation